front/app/api/campaigns/keys.ts

```diff
@@ -0,0 +1,21 @@
+import { QueryKeys } from 'utils/cl-react-query/types';
+import { QueryParameters } from './types';
+
+const baseKey = { type: 'campaign' };
+
+const campaignsKeys = {
+  all: () => [baseKey],
+  lists: () => [{ ...baseKey, operation: 'list' }],
+  list: (parameters: QueryParameters) => [
+    { ...baseKey, operation: 'list', parameters },
+  ],
+  item: ({ campaignId }: { campaignId?: string }) => [
```

Comment:
```suggestion
  item: ({ campaignId }: { campaignId: string }) => [
```

I don't think campaignId can be undefined (it doesn't make sense to request a campaign if you have no id). Right?